pkg/cliplugins/kubestellar/get_log/cmd/cmd.go

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/client-go/util/homedir"
+)
+
+var (
+	kubeconfig   string
+	followFlag   bool
+	linesFlag    string
+	whichArg     string
+	kubectlFlags []string
+	ksNamespace  string
+)
+
+func New(streams genericclioptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get_log [flags] which",
+		Short: "Get the log output from a central KubeStellar or kcp process",
+		Args:  cobra.ExactArgs(1),
+		Run:   getLog,
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&kubeconfig, "kubeconfig", "c", "", "Path to the kubeconfig file")
+	flags.BoolVarP(&followFlag, "follow", "f", false, "Follow the log output")
+	flags.StringVarP(&linesFlag, "lines", "n", "+0", "Number of lines to display")
+	flags.StringArrayVar(&kubectlFlags, "kubectl-flag", []string{}, "Additional kubectl flags")
```

Comment:
These are not connected to anything.

A user of a kubectl plugin should not have to wrap the ordinary kubectl flags.

Reply:
true - going to remove this line